docs/se/guides/tracing.adoc

```diff
@@ -232,10 +249,10 @@ image::guides/12_tracing_span_detail.png[Span Details]
 
 === Tracing Across Services
 
-Helidon automatically traces across services, providing that the services use the same tracer, for example, the same instance of Jaeger.
-This means a single trace can include spans from multiple services and hosts.  OpenTracing uses a `SpanContext` to
+Helidon automatically traces across services if the services use the same tracer, for example, the same instance of Jaeger.
+This means a single trace can include spans from multiple services and hosts.  Helidon uses a `Span.context()` to
 propagate tracing information across process boundaries.  When you make client API calls, Helidon will
-internally call OpenTracing APIs to propagate the `SpanContext`. There is nothing you need to do in your application to make this work.
+internally call OpenTracing APIs to propagate the Span Context. There is nothing you need to do in your application to make this work.
```

Comment:
V4 still has a `SpanContext` interface `io.helidon.tracing`. I think we can continue to refer to it as in the original document. If you want to avoid referring to that type for some reason, "Helidon uses a `Span.context()`" is awkward wording; maybe just use "span context" here and a couple lines below (lower case with space between the words).

It does make good sense to change OpenTracing to Helidon as shown.

Reply:
Fixed :)